app/src/main/res/values/strings.xml

```diff
@@ -224,4 +224,6 @@
     <string name="generic_alert_title">Alert</string>
     <string name="generic_exception_message">Unknown Exception. Please contact Yona support</string>
     <string name="generic_mobile_number_format_error">Invalid Mobile Number. Please check the format</string>
+    <string name="yona_notification_content">Device app activity is monitored by Yona</string>
```

Comment:
Please make this "App usage is being monitored by Yona"

Reply:
Done.